wire/msggetcfilter_test.go

```diff
@@ -0,0 +1,293 @@
+// Copyright (c) 2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package wire
+
+import (
+	"bytes"
+	"github.com/davecgh/go-spew/spew"
+	"github.com/decred/dcrd/chaincfg/chainhash"
+	"io"
+	"reflect"
+	"testing"
+)
+
+// TestMsgGetCFilter tests the MsgGetCFilter API.
+func TestGetCFilter(t *testing.T) {
+	pver := ProtocolVersion
+
+	// Block 200000 hash.
+	hashStr := "000000000000007a59f30586c1003752956a8b55e6f741fd5f24c800cd5e5e8c"
+	blockHash, err := chainhash.NewHashFromStr(hashStr)
+	if err != nil {
+		t.Errorf("NewHashFromStr: %v", err)
```

Comment:
This should probably be `t.Fatalf` because otherwise the test will keep running and it'll end up crashing due to a nil `blockHash`.

The same with the rest.  Typically, you only want to use `t.Errorf` when the tests are in a loop and you intentionally want to continue to the next one after reporting the error.